srcpkgs/btrfs-backup/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'btrfs-backup'
+pkgname=btrfs-backup
+version=0.3.1
+revision=1
+wrksrc="${pkgname/-/_}-${version}"
+noarch=yes
+build_style=python3-module
+pycompile_module="${pkgname/-/_}"
+hostmakedepends="python3-setuptools"
+depends="python3-setuptools"
```

Comment:
https://github.com/efficiosoft/btrfs-backup#requirements

`btrfs-progs` is required, obviously... but it's already part of `base-system` so I don't think we have to add it explicitly.